certified-connectors/AzureDigitalTwins/apiDefinition.swagger.json

```diff
@@ -343,7 +343,7 @@
         "operationId": "GetModelById",
         "parameters": [
           {
-            "$ref": "#/parameters/twinid"
+            "$ref": "#/parameters/modelid"
           },
           {
             "name": "includeModelDefinition",
```

Comment:
@sankumrav The 'x-ms-summary' property is required, and the 'description' property is required.

Reply:
@Amjed-Ayoub - The reference parameter modelid has the required information.